worker.go

```diff
@@ -68,7 +68,7 @@ func startWorkers(fileName string, nbWorkers int, env map[string]string) error {
 				r, err = NewRequestWithContext(
 					r,
 					WithRequestDocumentRoot(filepath.Dir(absFileName), false),
-					WithRequestEnv(env),
+					WithRequestPreparedEnv(env),
```

Comment:
Changing env after the worker starts will cause a race condition. You need to do this before starting the workers because all workers share the same env.

Reply:
I'm not sure what you mean. I don't change the current behavior, do I?

Reply:
> I don't change the current behavior, do I?

Nope. Git diff was confusing until I looked at it in split mode. It looked like you changed the method `WithRequestEnv` (which you did) to now cause side-effects via `PrepareEnv` but didn't change this (or vice-versa).